### PR TITLE
Exercise

### PR DESCRIPTION
**Pull request by kinuesantos**

### CODE REVIEW

File: lesson4/public/tickets.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  How many tickets you need:<input type='text' id='syatick'/>
+  <input type='submit' id='syaButton'/>
+  <div>
+    <pre id= 'displaysya'></pre>
+  </div>
+  <script>
+    //send the # of tickets bought
+    const button = document.getElementById('theButton');
+    button.onclick = () =>{
+    const quantity = document.getElementById('#Tickets').value;
```
Comment (songz) [inline lesson4/public/tickets.html:25]: indent

Comment (songz) [inline lesson4/public/tickets.html:25]: don't put `#` in your id name. its confusing.

File: lesson4/public/tickets.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    show2.innerHTML = xhttp2.responseText;
+    }
+  }
+xhttp2.send()
+}
+
+//Function to update the number of Tickets
+
+const update2 = () =>{
+  const xhttp3 = new XMLHttpRequest();
```
Comment (songz) [inline lesson4/public/tickets.html:76]: you can put all these into unseg function. This way, you don't need to have 2 setIntervals in your code.

File: lesson4/public/tickets.html (hunk 1 above)
Comment (kinuesantos) [reply lesson4/public/tickets.html:25]: done

File: lesson4/public/tickets.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        const show1 = document.getElementById('display2');
+        show1.innerHTML = xhttp.responseText;
+      }
+    }
+    xhttp.send();
+    }
+    }
+    
+    //get update every second
+    const unseg= ()=>{
```
Comment (chocobrownie) [inline lesson4/public/tickets.html:44]: I don't know what does unseg mean

File: lesson4/public/tickets.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+//Function to update the number of Tickets
+
+const update2 = () =>{
+  const xhttp3 = new XMLHttpRequest();
+  xhttp3.open('GET', `http://sz.llip.life/tickets`);
+  xhttp3.onreadystatechange = () =>{
+    if(xhttp3.responseText){
+      const current2 = document.getElementById('displaysya2');
+      console.log(current2);
```
Comment (chocobrownie) [inline lesson4/public/tickets.html:81]: remove console.log

File: lesson4/tickets.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+let count = 100;
+
+app.get('/santos',(req,res)=>{
+  const data = req.query.number;
+  if(data > count){
+ res.send('Not enough tickets')
+  }else{
+    count = count - data;
+    res.send('You bought: ' + (data + '') + 'tickets');
```
Comment (chocobrownie) [inline lesson4/tickets.js:14]: you don't need data + '', and you don't need brackets. Try to rethink the mechanism why number + '' converts it. Don't just memorize the code and 'paste' it

File: lesson4/public/tickets.html [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+      xhttp1.send();
+      xhttp1.onreadystatechange = () =>{
+        if(xhttp1.responseText){         
+          const show = document.getElementById('display');
+          show.innerHTML = xhttp1.responseText;
+        }
+      }
+    }
+    setInterval(unseg, 1000);
+//SYA code
```
Comment (chocobrownie) [inline lesson4/public/tickets.html:56]: Good job for adding comments for readability

File: lesson4/public/tickets.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
```
Comment (chocobrownie) [inline lesson4/public/tickets.html:5]: If you have the title tag, use it. Or else remove it

**Pull request merged**